src/app/views/invoices/modals/adjustment-modal.less

```diff
@@ -1,5 +1,8 @@
-#adjustment-modal {
-  .col-sm-1 .nopadding {
+.adjustment-modal {
+  .col-sm-1.nopadding {
     padding-left: 0 !important;
   }
+  .popup-currency-height {
+    line-height: 30px;
```

Comment:
Lets move this to `.col-sm-1.nopadding` block as well. Doesnt seem necessary to make a new class here

Reply:
Feedback has been incorporated.